aws_cli/tasks/main.yml

```diff
@@ -2,4 +2,7 @@
 # tasks file for aws_cli
 
 - name: Install awscli
-  pip: name=awscli
+  pip:
```

Comment:
Ditto my comment for Ansible - state in the README that it will be installed via pip